tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -23,6 +23,10 @@ apt-get install -y \
     build-essential \
     curl \
     git \
+    gfortran \
```

Comment:
I think using --system-site-packages is a good idea. I will incorporate it into this PR and make other related, necessary changes in Dockerfiles to make it work, if that's okay with you.